Sources/Testing/EntryPoints/EntryPoint.swift

```diff
@@ -452,17 +452,17 @@ extension Event.ConsoleOutputRecorder.Options {
     return result
   }
 
-  /// Whether or not the current process's standard error stream is capable of
-  /// accepting and rendering ANSI escape codes.
+  /// A Boolean value that indicates whether the current process's standard 
```

Comment:
This is a function, not a boolean value, so it should probably be "Verb whether or not blah blah".